package.json

```diff
@@ -25,6 +25,7 @@
     "ignore": "^3.3.3",
     "jest-docblock": "21.1.0",
     "jest-validate": "21.1.0",
+    "mem": "^1.1.0",
```

Comment:
Should this be without the `^` to be consistent with (most) other dependencies?

Reply:
Yeah. Pretty sure I used `yarn add mem` which should look at `.yarnrc`... This PR seems to be full of tools not looking at their config files...